api/config_entry_gateways_test.go

```diff
@@ -195,23 +195,6 @@ func TestAPI_ConfigEntries_TerminatingGateway(t *testing.T) {
 	require.NotNil(t, wm)
 	require.NotEqual(t, 0, wm.RequestTime)
 
-	// web is associated with the other gateway, should get an error
```

Comment:
intentional removal?

Reply:
Yes, intentional removal. This was again testing that a service could only be associated to one gateway. We have removed that restriction and validate it works correctly in the `state/catalog_test.go` tests.